src/org/folio/rest/Okapi.groovy

```diff
@@ -173,19 +172,30 @@ class Okapi extends GeneralParameters {
      * @param admin_user
      * @return
      */
-    void reindexId(tenant, jobId) {
+    void reindexid(tenant, jobid) {
         auth.getOkapiToken(tenant, tenant.admin_user)
-        String url = okapiUrl + "/instance-storage/reindex/${jobId}"
+        String url = okapiUrl + "/instance-storage/reindex/${jobid}"
         ArrayList headers = [
             [name: 'Content-type', value: "application/json"],
             [name: 'X-Okapi-Tenant', value: tenant.getId()],
             [name: 'X-Okapi-Token', value: tenant.getAdmin_user().getToken() ? tenant.getAdmin_user().getToken() : '', maskValue: true]
         ]
-        def res = http.getRequest(url, headers)
-        if (res.status == HttpURLConnection.HTTP_OK) {
-            logger.info(tools.jsonParse(res.content))
-        }else {
-            throw new AbortException("not possible check id reindex." + http.buildHttpErrorMessage(res))
+        steps.timeout(15) {
+            while (true) {
+                def res = http.getRequest(url, headers)
+                if (res.status == HttpURLConnection.HTTP_OK) {
+                    if (tools.jsonParse(res.content).status == "Ids published") {
+                        return true
+                        logger.info("reindex successufully completed")
+                        break
+                    } else if (tools.jsonParse(res.content).status != "Ids published") {
+                        logger.info("Waiting timeout, haven't status: Ids published yet." + http.buildHttpErrorMessage(res))
+                        steps.sleep(10)
+                    } else {
```

Comment:
looks like this "else" must be for "res.status == HttpURLConnection.HTTP_OK" if.